decidim-core/app/types/decidim/core/has_locale_input_filter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    module HasLocaleInputFilter
```

Comment:
This artifact manages localized input filters, no?
```suggestion
    module HasLocalizedInputFilter
```